.github/workflows/pre_integration_checks-juju3.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
I simplyfied the workflow skipping the libs check, but of course we can unify this file with the existing `pre_integration_checks,yaml` (with `LIBJUJU_VERSION_SPECIFIER` passed)

Reply:
I'll actually do it on the next round of changes on the PR:-)